src_assets/common/assets/web/configs/tabs/General.vue

```diff
@@ -127,6 +127,16 @@ function removeCmd(index) {
         &plus; {{ $t('config.add') }}
       </button>
     </div>
+
+    <!-- Notify Pre-Releases -->
+    <div class="mb-3">
+        <label for="notify_pre_releases" class="form-label">{{ $t('config.notify_pre_releases') }}</label>
+        <select id="notify_pre_releases" class="form-select" v-model="config.notify_pre_releases">
+            <option value="disabled">{{ $t('_common.disabled') }}</option>
+            <option value="enabled">{{ $t('_common.enabled') }}</option>
+        </select>
+        <div class="form-text">{{ $t('config.notify_pre_releases_desc') }}</div>
```

Comment:
How about, stable, beta and none? This way if someone on beta, want to start receiving the next stable whenever it launches, they can too

Reply:
As you clarified, we aways shows stable updates, so there's no need to change channel.